sips/completed/_posts/2016-06-25-trailing-commas.md

```diff
@@ -3,31 +3,33 @@ layout: sip
 disqus: true
 title: SIP-27 - Trailing Commas
 
-vote-status: accepted 
+vote-status: under review
 vote-text: The following proposal needs to be updated, since only the specialized case version (with new lines) has been accepted. For more information, check the <a href="http://docs.scala-lang.org/sips/minutes/sip-20th-september-minutes.html">minutes</a>.
 ---
 
+// TODO: Move from sips/completed to sips/pending
+
```

Comment:
I'm removing the TODO and moving the document in https://github.com/scala/scala.github.com/pull/631, which can be merged after this is merged.